hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstant.java

```diff
@@ -83,7 +85,7 @@ public static String getTimelineFileExtension(String fileName) {
       return fileName.substring(matcher.group(1).length());
     }
 
-    return "";
+    return StringUtils.EMPTY_STRING;
   }
```

Comment:
We better define a custom constant here named: `EMPTY_INSTANT` instead of using a general constant.

Reply:
Can you address this comment ？